diff --git a/node_modules/react-waypoint/es/index.js b/node_modules/react-waypoint/es/index.js
index ed6c50b..72d9bc6 100644
--- a/node_modules/react-waypoint/es/index.js
+++ b/node_modules/react-waypoint/es/index.js
@@ -2,7 +2,8 @@ import _inheritsLoose from '@babel/runtime/helpers/esm/inheritsLoose';
 import { addEventListener } from 'consolidated-events';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { isForwardRef } from 'react-is';
+import reactIs from 'react-is';
+const { isForwardRef } = reactIs;
 
 /**
  * Attempts to parse the offset provided as a prop as a percentage. For
